index.html

```diff
@@ -225,18 +233,22 @@ <h4 class="item_name">Skills</h4>
       </li>
     </ul>
     <footer class="portofolio_footer">
-      <div class="contact_me_section">
-        <h1 class="contact_form_title">
+      <div class="contact_me_section"> 
+        <div class="completed_form">
+          <h1 class="contact_form_title">
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-        </h1>
-        <form class="contact_form" action="https://formspree.io/f/mknegjnj" method="POST">
-          <input type="text" name="clientName" class="form_input" placeholder="Full Name" maxlength="30" required>
-          <input type="email" class="form_input" name="clientEmail" placeholder="Email address" required>
-          <textarea id="contact_text_area" class="form_input" name="clientMessage" placeholder="write your message here" maxlength="500"
-            required></textarea>
-          <button class="submit_button" type="submit">Get in touch</button>
-        </form>
-        <ul class="main_section_social">
+          </h1>
+          <form class="contact_form" action="https://formspree.io/f/mknegjnj" method="POST">
+            <input type="text" name="clientName" class="form_input" placeholder="Full Name" maxlength="30" required>
+            <input type="email" class="form_input" name="clientEmail" placeholder="Email address" required>
+            <textarea id="contact_text_area" class="form_input" name="clientMessage" placeholder="write your message here" maxlength="500"
+              required></textarea>
+            <button class="submit_button" type="submit">Get in touch</button>
+          </form>
+        </div> 
```

Comment:
- Kindly make sure you use the correct background for your about page and the contact page to make your design stick to the Figma design as required. Here I provide the link for the Figma design so that you can even copy those background images. your design should look to the image show below:+1: :grinning:  :grinning:  :grinning:
 
## Figma design 

![image](https://user-images.githubusercontent.com/95297251/186788577-eaa3f0f0-f11b-44d2-84e3-0f1a763ab325.png)